feature-group-definitions/slot.dist.yml

```diff
@@ -0,0 +1,32 @@
+# Generated from: slot.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: <slot>
+spec: https://html.spec.whatwg.org/multipage/scripting.html#the-slot-element
+usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/1898
+status:
+  baseline: low
+  baseline_low_date: 2023-03-27
+  support:
+    chrome: "86"
```

Comment:
This should probably be marked as supported from earlier:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLSlotElement#browser_compatibility